alfred-telemetry-solr/README.md

```diff
@@ -105,77 +105,77 @@ At the moment following metrics are included, for each core tracked by solr:
 
 | Name                                                                  |
 | ----------------------------------------------------------------------|
-| alfresco_nodes{core=<core>,feature="Approx transactions remaining"}   |
-| alfresco_nodes{core=<core>,feature="TX lag"}                          |
-| alfresco_nodes{core=<core>,feature="Last Index TX Commit Time"}       |
-| alfresco_acls{core=<core>,feature="Approx change sets remaining"}     |
-| alfresco_acls{core=<core>,feature="Change Set Lag"}                   |
-| alfresco_acls{core=<core>,feature="Last Index Change Set Commit Time"}|
+| alfresco.nodes{core=<core>,feature="Approx transactions remaining"}   |
```

Comment:
can these be semantically added with ` alfresco.nodes{core=<core>,feature="Alfresco Stated in Index"}` ?

Reply:
I believe it's 1 state per tracker => semantically they would belong to both alfresco.nodes and alfresco.acls (impossible)